src/interface/PremiumLoginPanel.tsx

```diff
@@ -98,7 +98,7 @@ const PremiumLoginPanel = () => {
   return (
     <div className="panel">
       <div className="panel-body" style={{ padding: '0 15px', position: 'relative' }}>
-        <TransitionGroup classNames="logged-in" timeout={1000}>
+        <TransitionGroup className="logged-in" timeout={1000}>
```

Comment:
pretty sure this is not a correct change: https://github.com/WoWAnalyzer/WoWAnalyzer/commit/ba5c2dd07204082ffbe74d491a0055b42a67aa77

Reply:
I was getting this error in Chrome, which lead to the change:
![image](https://github.com/WoWAnalyzer/WoWAnalyzer/assets/1672786/e11a9383-759a-4a52-bf67-a9a7a8123725)